### PR TITLE
Update running-gatewayd.md to handle Docker host connection for different setups

### PR DESCRIPTION
**Pull request by sinadarbouy**
## Description
This PR addresses the issue of setting the Docker host variable when running on Docker Desktop. Due to the networking implementation in Docker Desktop, the `docker0` bridge interface is not present on the host. Instead, we can use the DNS name (https://docs.docker.com/desktop/networking/#use-cases-and-workarounds)

## Changes Made
- Added instructions for setting the `DOCKER_HOST` variable to `host.docker.internal` when using Docker Desktop.
- Removed the extra definition of DOCKER_HOST in step 5 because it is already defined in step 1.

## Context
When running on Docker Desktop, the absence of the docker0 bridge interface on the host can lead to confusion regarding how to correctly set up the Docker host variable.

### CODE REVIEW

File: getting-started/running-gatewayd.md [hunk 1]
```diff
@@ -120,7 +130,6 @@ Running GatewayD will produce the following log output, which means that Gateway
 GatewayD is running on the host, while PostgreSQL is running inside a container. You can run the following command to test it. Notice that `${DOCKER_HOST}` holds the IP address of the host machine, that is accessible from inside the container.
 
 ```bash
-DOCKER_HOST=$(ip addr show docker0 | grep inet | grep -v inet6 | awk -F' ' '{ print $2 }' | sed 's/\/16//g')
```
Comment (mostafa) [inline getting-started/running-gatewayd.md:123]: @sinadarbouy How about using this one-liner instead? It checks whether the `docker0` interface exists and acts accordingly. Then we can also remove the above explanations.
```bash
DOCKER_HOST=$(if ip addr show docker0 > /dev/null 2>&1; then ip addr show docker0 | grep inet | grep -v inet6 | awk -F' ' '{ print $2 }' | sed 's/\/.*//'; else echo "host.docker.internal"; fi);
```

**Review by mostafa — COMMENTED** [review]
@sinadarbouy Thanks for your contribution!

I made a comment to simplify your change.

Consider running `bash generate_last_modified_data.sh` and committing the update timestamps in all files.

File: getting-started/running-gatewayd.md [hunk 2]
```diff
@@ -29,10 +29,20 @@ Use the following command to start a PostgreSQL database server inside a contain
...
+```
+
+**if there is no `docker0` bridge (using Docker Desktop)**
+Use the DNS name `host.docker.internal`, which resolves to the internal IP address used by the host.
+```bash
+DOCKER_HOST="host.docker.internal"
+```
+
+Run the following command to test your database:
+```bash
```
Comment (mostafa) [inline getting-started/running-gatewayd.md:45]: @sinadarbouy How about using this one-liner instead? It checks whether the `docker0` interface exists and acts accordingly. Then we can also remove the above explanations.
```bash
DOCKER_HOST=$(if ip addr show docker0 > /dev/null 2>&1; then ip addr show docker0 | grep inet | grep -v inet6 | awk -F' ' '{ print $2 }' | sed 's/\/.*//'; else echo "host.docker.internal"; fi);
```

Comment (sinadarbouy) [reply getting-started/running-gatewayd.md]: :+1: Good idea!

**Review by mostafa — APPROVED** [review]

**Pull request merged**